public/javascripts/SVLabel/src/SVLabel/task/Task.js

```diff
@@ -342,6 +343,16 @@ function Task (geojson, currentLat, currentLng) {
         return _geojson.features[0].properties.completion_count;
     };
 
+    this.getStreetPriority = function () {
+        return _geojson.features[0].properties.priority;
+    };
+
+    // Returns an integer in the range 0 to n-1, where larger n means higher priority.
+    this.getStreetPriorityDiscretized = function() {
+        var n = 4;
+        return Math.min(Math.floor(_geojson.features[0].properties.priority / (1 / n)), n - 1);
```

Comment:
What is the logic for this?

Reply:
If you wanted to split the range [0,1] into n = 4 ranges, each sub-range has a length of 1 / n = 1 / 4 = 0.25. Then you get the discretized order by taking floor(priority / 0.25), which brings [0,0.25) -> 0, [0.25,0.5) -> 1, [0.5,0.75) -> 2, [0.75,1) -> 3, and 1 -> 4. But we really want [0.75-1] -> 3, so instead of floor(priority / (1 / n)), we have min(floor(priority / (1 / n)), n - 1).

Reply:
Cool. Thanks for the clarification. I am adding this in the comment.